src/BIAnalyticsManagementService/CreateParticipantScreeningEpisode/CreateParticipantScreeningEpisode.csproj

```diff
@@ -10,6 +10,8 @@
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.19.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Http" Version="3.0.13" />
```

Comment:
I don't think we need this package any more

Reply:
Hi Alex, I tried removing them and got build errors

Reply:
removed it.